tools/AzureRM.Example.psm1

```diff
@@ -23,12 +23,15 @@ $FilteredCommands = %DEFAULTRGCOMMANDS%
 if ($Env:ACC_CLOUD -eq $null)
 {
     $FilteredCommands | ForEach-Object {
-        $global:PSDefaultParameterValues.Add($_,
-            {
-                $context = Get-AzureRmContext
-                if (($context -ne $null) -and $context.ExtendedProperties.ContainsKey("Default Resource Group")) {
-                    $context.ExtendedProperties["Default Resource Group"]
-                } 
-            })
+        if ($global:PSDefaultParameterValues.Contains($_))
```

Comment:
Don't you want to add the alias ony if it *is not* already there?

Reply:
also, need profile changelog update

Reply:
Good catch, had that locally but hadn't committed before creating the pull request.  Added a changelog entry for every module with a psm1 file that could exhibit this bug.